docs/part1.md

```diff
@@ -33,18 +33,490 @@ The landing NOMAD Entries page is a very intuitive and easy-to-use Graphical Use
+
+System specifications are found under "Material".
+Since we know the composition of our system, **click on "Elements / Formula"** and **select the matching elements** in the periodic table.
+The number of database entries containing the other elements too is communicated via the color code.
+Those that are already **filtered out will be gray**, just as you saw when opening the side pane.
+This is the filter stacking mentioned before, showing that Gromacs users typically work on organic systems.
+You can see both chips on the left.
+Note how there is an "and" in between.
+While **filters** between side menu options stack, those **within a side pane each have their own logic**.
+We are looking for pure water, so O and H suffice. <!-- Shame how we don't have a subgroup search yet -->
+**Remove all data with additional elements** by selecting "only compositions that exclusively contain these atoms".
+
+!!! success
+    You are left with 2 MD calculations in the entries list.
+    Clicking on either, will fold out a quick summary.
+    **Clicking on the arrow** (`->`) will bring you to an _Overview page_.
+    More on that in [Part II](part2.md).
+    By **clicking through on "FILES"**, you will get an overview of the uploaded files.
+    These for sure contain output, but often also input.
+    Both could serve as great starting points for deploying your own calculations.
+
+Actually, the **exact historical order** in which we applied the filters **does not matter**.
+I.e. the same set of filters will yield the same search result.
+This insight comes with a small caveat: the statistics that we see as we are stacking up our filters will behave differently.
+These filters are like small questions or details that we use construct an overall query.
+If we take a **different route**, we will see **different intermediate results** and this could lead us to **change our final formulation**.
+
+To see this effect in action, rewind back to where we were choosing our priorities and let us go with the latter instead.
+Just **click the cross (`x`) at the top of the filter menu**, and **reapply the filters** for Gromacs and the composition.
+Just with these settings, the entries list is so short, that we could **check the specific calculations** even **by hand**!
+
+!!! success
+    Glossing over the remaining 6 results, you discover that Sebastian Baugmart has 2 other MD simulations which were not classified as such. <!-- @Joseph is this a bug? -->
+    We also find 2 more Geometry Optimizations of both systems.
+    While these do not exactly match our initial objective, they could still provide us with valuable insights.
+    They also inform us of the overarching workflow used by both authors, i.e. first relax the system, then run MD under various constraints. <!-- @Joseph: thermostats? -->
+
+### Scenario 2 - Data science
```

Comment:
I flagged 2 possible places using placeholders of the format `<!-- placeholder note for Joseph option A/B -->`.
I also left a suggestion at placeholder A, in case you don't get to revisit this in time.